frontend/src/components/AccountReceiveTableWithdrawConfirmation.vue

```diff
@@ -191,9 +186,9 @@ export default defineComponent({
     const customFeeInWei = computed(() => {
       return gasLimit.value.mul(customGasInWei.value);
     });
-    const formattedCustomFeeEth = computed(() =>
-      round(formatUnits(customFeeInWei.value, decimals), ethDisplayDecimals)
-    );
+    const formattedCustomFeeEth = computed(() => {
+      return humanizeTokenAmount(customFeeInWei.value, props.activeFee.token);
+    });
```

Comment:
nit, but my personal preference (and therefore frontend convention in this repo) is to keep this on one line since it's <120 chars long (not sure why it wasn't like that before)

Reply:
Really it auto-breaks at 80 chars. Split screen vim has to wrap after 95 chars

120 character lines are inhumane. I will die on this hill.

Reply:
use a wider monitor it's 2022! 😛